components/eventing-controller/README.md

```diff
@@ -55,14 +41,13 @@ This section explains how to use the Eventing Controller. It expects the followi
     | WEBHOOK_TOKEN_ENDPOINT | The Kyma public endpoint to provide Access Tokens.                              | BEB     |
     | DOMAIN                 | The Kyma cluster public domain.                                                 | BEB     |
     | NATS_URL               | The URL for the NATS server.                                                    | NATS    |
-    | APP_LOG_FORMAT         | The format of the Application logs.                                             | Both    |
-    | APP_LOG_LEVEL          | The level of the Application logs.                                              | Both    |
```

Comment:
We need those env vars. Please revert the deletion.

Reply:
Will do. Strange, why had they being removed. 🤔 Possibly git error on my side.